nex/faq.md

```diff
@@ -0,0 +1,70 @@
+# Nex FAQ
+Frequently Asked Questions about the NATS Execution Engine
+
+### General
+
+* [What is Nex?](faq.md#what-is-nex)
+* [Where can I find Nex?](faq.md#where-can-i-find-nex)
+* [Who maintains Nex?](faq.md#who-maintains-nex)
+* [Is Nex free?](faq.md#is-nex-free)
+
+### Running Workloads
+
+* [What workload types can I run?](faq.md#what-workload-types-can-i-run)
+* [What is a root file system?](faq.md#what-is-a-root-file-system)
+
+### Technical Details
+
+* [What are namespaces?](faq.md#what-are-namespaces)
+* [Are my workload configurations secure?](faq.md#are-my-workload-configurations-secure)
+* [What is the difference between run and devrun?](faq.md#what-is-the-difference-between-run-and-devrun)
+
+## General
+
+### What is Nex?
+Nex is an open source, lightweight execution engine that can be run in addition to and alongside NATS. Nex nodes form a universal pool into which you can deploy services and functions in JavaScript, WebAssembly, and any language that compiles to native 64-bit Linux binaries.
+
+### Where Can I Find Nex?
+The code for Nex can be found in the appropriate [Synadia](https://github.com/synadia-io/nex) repository. It is an open source project that is supported by Synadia, the creators of NATS.
+
+### Who Maintains Nex?
+The primary maintainers of the Nex code are employees of Synadia, though more maintainers are always welcome.
+
+### Is Nex Free?
+Yes, all of the code in the Nex repository is free and licensed under the **Apache-2.0** license.
+
+## Running Workloads
+
+### What workload types can I run?
+Nex supports two categories of workloads: **services** and **functions**. Services are long-lived while functions are short-lived and triggered on demand by some stimulus.
+
+Services can be any statically compiled 64-bit Linux binary and functions can be either a **WebAssembly** module or a **JavaScript** exported function. We are exploring possible support for running OCI images as services.
+
+### What is a root file system?
+Nex uses [firecracker](https://firecracker-microvm.github.io/) to launch small, very fast and very light weight virtual machines. These machines each run an agent that manages the workload in that machine as well as communications with the Nex node.
+
+The root file system is essentially a snapshot of the operating system disk image that will be used when bringing up the virtual machine. It's like a docker layer, but much more efficient.
+
+## Technical Details
+
+### What are namespaces?
+A namespace is a unit of multi-tenancy. All workloads are scoped to a namespace. If you deploy the `echoservice` workload within the `default` namespace and also deploy it within the `sample` namespace, they will not be treated the same by Nex. 
+
+**Note** - there's nothing Nex can do from preventing your code in both of those services from potentially overlapping, clustering, etc.
```

Comment:
Would it be more clear to state that namespaces are not a form of isolation?

Reply:
but they are a form of isolation.. we just don't force network isolation. not sure the best way to convey that.